src/main/java/xyz/aadev/generitech/Reference.java

```diff
@@ -58,6 +58,7 @@ private Reference() {
         public static final int PULVERIZER = 0;
         public static final int FURNACE = 1;
         public static final int GENERATOR_GUI = 2;
+        public static final int POWERSTORAGE_GUI = 3;
     }
 
     public static final class Commands {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1118)